types/three/examples/jsm/loaders/VOXLoader.d.ts

```diff
@@ -12,3 +12,11 @@ export class VOXLoader extends Loader {
     loadAsync(url: string, onProgress?: (event: ProgressEvent) => void): Promise<object[]>;
     parse(data: ArrayBuffer): object[];
 }
+
+export class VOXMesh extends Mesh {
+    constructor(chunk: object);
+}
+
+export class VOXDataTexture3D extends DataTexture3D {
+    constructor(chunk: object);
```

Comment:
Can we provide a type alias (e.g., `type Chunk = object`) that can be shared between `VOXLoader`, `VOXMesh`, and `VOXDataTexture3D` just to make it clear that they're connected?

Reply:
I actually realised I could piece it together using the `load` function, so now its a lot better